api-reference/beta/toc.yml

```
@@ -13525,6 +13525,124 @@ items:
+            - name: Get a Deleted Workflow 
+              href: api/identitygovernance-deleteditemcontainer-get.md  
+            - name: Permanently Delete a Workflow
+              href: api/identitygovernance-deletedItemcontainer-delete.md    
+            - name: Restore
+              href: api/identitygovernance-workflow-restore.md
+            - name: Activate
+              href: api/identitygovernance-workflow-activate.md
+            - name: List Tasks
+              href: api/identitygovernance-workflow-list-task.md
+            - name: Get Task
+              href: api/identitygovernance-workflow-task-get.md  
+            - name: Versioning
+              items:
+              - name: List
+                href: api/identitygovernance-list-workflowversion.md
+              - name: Get
+                href: api/identitygovernance-workflowversion-get.md
+              - name: Create
+                href: api/identitygovernance-workflow-createnewversion.md 
+              - name: List Tasks
+                href: api/identitygovernance-workflowversion-list-tasks.md
+              - name: Get Task
+                href: api/identitygovernance-workflowversion-task-get.md      
+            - name: Reporting
+              items:
+              - name: Overview
+                href: resources/identitygovernance-lifecycleworkflows-reporting-overview.md
+              - name: User Processing Result
+                href: resources/identitygovernance-userprocessingresult.md
+                items:
+                - name: List
+                  href: api/identitygovernance-workflow-list-userprocessingresults.md
+                - name: Summary
+                  href: api/identitygovernance-userprocessingresult-summary.md  
+              - name: Task Reports
+                href: resources/identitygovernance-taskreport.md 
+                items:
+                - name: List
+                  href: api/identitygovernance-taskreport-list.md
+                - name: Summary
+                  href: api/identitygovernance-taskreport-summary.md
+                - name: List Task Processing Results
+                  href: api/identitygovernance-taskreport-taskprocessingresult-list.md    
+              - name: Workflow runs
+                href: resources/identitygovernance-run.md
+                items:
+                - name: List
+                  href: api/identitygovernance-workflow-list-run.md
+                - name: Get
+                  href: api/identitygovernance-run-get.md
+                - name: Summary
+                  href: api/identitygovernance-run-summary.md  
+                - name: List User processing Results
+                  href: api/identitygovernance-run-list-userprocessingresults.md
+                - name: Get User processing Results
+                  href: api/identitygovernance-run-userprocessingresult-get.md     
+              - name: Task Processing Result
```

Also add LIST (identitygovernance-taskreport-list-taskprocessingresults.md) and GET (identitygovernance-taskprocessingresult-get.md). The resource file should also list these methods.
```suggestion
              - name: Task processing result
```